index.html

```diff
@@ -33,5 +42,139 @@ <h3>I'm a software Developer, </h3>
         </ul>
       </div>
     </section>
+    <section class="section-two">
+      <h1>My Recent Works</h1>
+        <img class="first-divider" src="./final-images/Vector.png" alt="a divider">
+        <img class="yoga-image" src="./final-images/Img Placeholder.png" alt="a lady doing yoga">
+        <h2>Multi-Post Stories</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+          has been the industry's standard dummy text ever since the 1500s, when an unknown printer took
+          a standard dummy text.
+        </p>
+        <ul class="stack-holder">
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+        <button class="see-project"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
```

Comment:
AS mention previously the default style for the button looks good! However, to enhance the user experience and provide visual feedback, it's important to add styles for the hover and active states of the button as well. These styles should match the design specifications and provide clear indications to the user when interacting with the button.
![image](https://github.com/prince-rutagengwa/mobile-portfolio/assets/55840999/677f4c2b-2b7b-4820-969f-1cfd0519524a)

Please review the design guidelines and incorporate the appropriate styles for the hover and active states of the button. This will ensure a consistent and visually pleasing user interface. Keep up the great work! 👍🎉